src/annotations/lang.js

```diff
@@ -1,3 +1,37 @@
+import React from 'react'
+
+// “Translate” icon from the Material Design icon library
+const MDTranslate = (
```

Comment:
I wonder if we should call it `MarkdownTranslateIcon` or `IconTranslate` or something else that makes it slightly more explicit?

Reply:
`MD` stands for Material Design in that case, but I would agree. Hold on.